Package.swift

```diff
@@ -18,16 +18,6 @@
 
 import PackageDescription
 
-var listDependencies: [Package.Dependency] = [
-    .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", from: "1.7.1"),
```

Comment:
I think we do still want to log errors, but this import should be LoggerAPI instead.